app/modules/app/hooks/useSearchQueryParams.ts

```diff
@@ -0,0 +1,129 @@
+import { useEffect, useState } from "react";
```

Comment:
Why do we place this file under hooks and not helpers? what is the difference? when do we use one the other? I ask because I'm doing something similar for authorization